pkg/kn/config/config.go

```diff
@@ -74,8 +74,6 @@ func (c *config) LookupPluginsInPath() bool {
 	if viper.IsSet(deprecatedKeyPluginsLookupInPath) {
 		return viper.GetBool(deprecatedKeyPluginsLookupInPath)
 	} else {
-		// If legacy branch is removed, switch to setting the default to viper
-		// See TODO comment below.
```

Comment:
This change should be reverted, since the PR isn't removing the `--lookup-path` flag.